### ISSUE TITLE
PDB file is not canceled occupancy

### ISSUE THREAD

**Issue by wind-Blow-ing** [cheat-engine/cheat-engine#1603]
When I use VS to run the code and use the cheat engine to open the process, when the process ends, I cannot change the code to run. It prompts that the pdb file is occupied

**Comment by cheat-engine** [comment]
are you using the latest version of ce?

if the process has closed it should release the pdb

**Comment by wind-Blow-ing** [comment]
Yes, the latest 7.2 version will be released only when CE is closed!

**Comment by wind-Blow-ing** [comment]
[](https://cdn.discordapp.com/attachments/841535303068155914/848942463276417045/unknown.png)

**Comment by wind-Blow-ing** [comment]
![unknown](https://user-images.githubusercontent.com/63902744/120214369-e8c80a80-c223-11eb-86af-b5df2bd0e072.png)

**Comment by Chaoses-Ib** [comment]
I have the same problem with CE 7.2. I compiled a project and open CE to debug it. After the process had exited, I went to recompile the project and got this:
> LINK : fatal error LNK1201: error writing to program database 'C:\Test\out\main.pdb'; check for insufficient disk space, invalid path, or insufficient privilege

I used a tool to check CE's handles and found that CE holds 4 file handles of the PDB file. Even when I switched to another process in CE, they are still not released.

**Comment by Chaoses-Ib** [comment]
This bug occurs every time I open a process with a PDB file in CE. And all the PDB files won't be released until CE is closed. It seems that there is a handle leak bug.
